cluster-scope/base/core/namespaces/debezium-ui/namespace.yaml

```diff
@@ -5,3 +5,6 @@ metadata:
     annotations:
         openshift.io/display-name: Debezium UI
         openshift.io/requester: debezium
+        op1st/project-owner:
+        op1st/onboarding-issue:
```

Comment:
I believe this is the onboarding issue: https://github.com/operate-first/support/issues/304